README.md

```diff
@@ -10,7 +10,8 @@ We have standardized on [Git LFS (Large File Storage)](https://git-lfs.github.co
 
 ## Models
 #### Read the [Usage](#usage-) section below for more details on the file formats in the ONNX Model Zoo (.onnx, .pb, .npz), downloading multiple ONNX models through [Git LFS command line](#gitlfs-), and starter Python code for validating your ONNX model using test data.
-#### INT8 models are generated by [Intel® Neural Compressor](https://github.com/intel/neural-compressor), read the [Introduction](https://github.com/intel/neural-compressor/blob/master/README.md) to know how to use it to quantize ONNX model.
```

Comment:
IMO, we probably can keep this line to show the relationship between Intel Neural Compressor and ONNX Model Zoo:
```
INT8 models are generated by [Intel® Neural Compressor](https://github.com/intel/neural-compressor)
```